packages/kbn-server-route-repository/src/decode_request_params.ts

```diff
@@ -36,8 +36,33 @@ export function decodeRequestParams<T extends RouteParamsRT>(
   const result = strictKeysRt(paramsRt).decode(paramMap);
 
   if (isLeft(result)) {
-    throw Boom.badRequest(PathReporter.report(result)[0]);
+    throw Boom.badRequest(formatErrors(result.left).join('|'));
   }
 
   return result.right;
 }
+
+export const formatErrors = (errors: t.Errors): string[] => {
```

Comment:
do you have an example of how this looks? Curious why we are writing our own, isn't there something built by the community? I do agree that the default formatting is teeeeerrrible so happy to see improvements here

Reply:
### after

```
{
    "statusCode": 400,
    "error": "Bad Request",
    "message": "Invalid value \"{\"groupBy\":[\"monitor.name\"]}\" supplied to \"body,list,groupBy\""
}
```

### Before

```
{
    "statusCode": 400,
    "error": "Bad Request",
    "message": "Invalid value {\"groupBy\":[\"monitor.name\"]} supplied to : { body: { list: Array<({ sloId: string, instanceId: string, timeWindow: ({ duration: Duration, type: \"rolling\" } | { duration: Duration, type: \"calendarAligned\" }), budgetingMethod: (\"occurrences\" | \"timeslices\"), objective: ({ target: number } & Partial<{ timesliceTarget: number, timesliceWindow: Duration }>), groupBy: (\"*\" | string | Array<(\"*\" | string)>), revision: number } & Partial<{ remoteName: string }>)> } }/body: { list: Array<({ sloId: string, instanceId: string, timeWindow: ({ duration: Duration, type: \"rolling\" } | { duration: Duration, type: \"calendarAligned\" }), budgetingMethod: (\"occurrences\" | \"timeslices\"), objective: ({ target: number } & Partial<{ timesliceTarget: number, timesliceWindow: Duration }>), groupBy: (\"*\" | string | Array<(\"*\" | string)>), revision: number } & Partial<{ remoteName: string }>)> }/list: Array<({ sloId: string, instanceId: string, timeWindow: ({ duration: Duration, type: \"rolling\" } | { duration: Duration, type: \"calendarAligned\" }), budgetingMethod: (\"occurrences\" | \"timeslices\"), objective: ({ target: number } & Partial<{ timesliceTarget: number, timesliceWindow: Duration }>), groupBy: (\"*\" | string | Array<(\"*\" | string)>), revision: number } & Partial<{ remoteName: string }>)>/0: ({ sloId: string, instanceId: string, timeWindow: ({ duration: Duration, type: \"rolling\" } | { duration: Duration, type: \"calendarAligned\" }), budgetingMethod: (\"occurrences\" | \"timeslices\"), objective: ({ target: number } & Partial<{ timesliceTarget: number, timesliceWindow: Duration }>), groupBy: (\"*\" | string | Array<(\"*\" | string)>), revision: number } & Partial<{ remoteName: string }>)/0: { sloId: string, instanceId: string, timeWindow: ({ duration: Duration, type: \"rolling\" } | { duration: Duration, type: \"calendarAligned\" }), budgetingMethod: (\"occurrences\" | \"timeslices\"), objective: ({ target: number } & Partial<{ timesliceTarget: number, timesliceWindow: Duration }>), groupBy: (\"*\" | string | Array<(\"*\" | string)>), revision: number }/groupBy: (\"*\" | string | Array<(\"*\" | string)>)/0: \"*\""
}
```

Reply:
i didn't write it, i update the path actually now to reuse from security io-ts utils